### ISSUE TITLE
Unable to install label-maker in Windows

### ISSUE THREAD

**Issue by SarthakTewari** [developmentseed/label-maker#132]
I'm getting the following error while installing label-maker using pip install label-maker.
Using Python 3.6.8.

Error message:  Command "python setup.py egg_info" failed with error code 1 in C:\Users\<username>\AppData\Local\Temp\pip-install-4nkv2dda\rasterio\

**Comment by drewbo** [comment]
@SarthakTewari are you able to install `rasterio` independently? It might work better if that's done separately. I haven't put together instructions yet for Windows so I'm not sure the best way to approach this

**Comment by SarthakTewari** [comment]
@drewbo Yes, I was able to install rasterio independently. However, "pip install label-maker" is still failing with the error mentioned above.

**Comment by Rub21** [comment]
@SarthakTewari , another option for you would be to use the label-maker docker container, take a look on https://github.com/developmentseed/docker-label-maker
